syntax = "proto3";

package com.embedd;

message EmbeddRequest {
    string content = 1;
    string model = 2;
}

message EmbeddResponse {
    repeated float vector = 1;
}

message EmbeddData {
    int64 id = 1;
    string content = 2;
    string model = 3;
    repeated float vector = 4;
}